app/scripts/modules/core/src/managed/ManagedResourceObject.tsx

```diff
@@ -56,6 +57,20 @@ const getResourceRoutingInfo = (
   return null;
 };
 
+const resourceStatusConfig: {
+  [status: string]: Pick<IStatusBubbleProps, 'appearance' | 'iconName'>;
+} = {
+  ACTUATING: { appearance: 'progress', iconName: 'mdActuating' },
+  CREATED: { appearance: 'info', iconName: 'mdCreated' },
+  DIFF: { appearance: 'info', iconName: 'mdDiff' },
+  CURRENTLY_UNRESOLVABLE: { appearance: 'error', iconName: 'mdError' },
+  ERROR: { appearance: 'error', iconName: 'mdError' },
+  PAUSED: { appearance: 'neutral', iconName: 'mdPaused' },
+  RESUMED: { appearance: 'progress', iconName: 'mdResumed' },
+  UNHAPPY: { appearance: 'error', iconName: 'mdUnhappy' },
+  UNKNOWN: { appearance: 'inactive', iconName: 'mdUnknown' },
```

Comment:
similar to `PAUSED`, this status uses the `warning` appearance on resources today. I think the same context around being 'probably not great' applies — really resources should _never_ enter this state, if they do it means there's almost certainly a bug we need to fix. It's not an error ('bad') but it's also something that's very important to draw attention to.

```suggestion
  UNKNOWN: { appearance: 'inactive', iconName: 'mdUnknown' },
```